Update defining-proficiency-levels.md (#328)
usds/SME-QA-Website: pages/phases/job-analysis/defining-proficiency-levels.md

sidenav: hiring-phases
phase: job-analysis
sub-phase-order: 050
intro: Define proficiency levels for each competency, determining what is needed to qualify in this hiring action.
---

<p class="usa-intro">
  {{ page.intro }}
</p>

### Setting the required proficiency level

Core competencies are the professional skills an applicant must possess from their first day in the position you're hiring for, while proficiencies gauge how skilled an applicant is with the given competency. The proficiency definitions should be consistent, involving these types of differences:

- **Complexity:** The applicant can manage straightforward workflows _-vs-_ can manage highly complex workflows.
- **Level of detail:** The applicant can explain a general concept _-vs-_ can discuss the concept in detail.
- **Scale of activity:** The applicant did something at a small organization _-vs-_ did something at a large organization.

> - **Seniority/independence:** The applicant did something as part of a group _-vs-_ lead the group that did something.

Reviewer: ```suggestion
- **Seniority/independence:** The applicant did something as part of a group _-vs-_ led the group that did something.
```

When defining proficiency levels, you have two options.

### Two levels

> Use two levels if your announcement is only for one GS grade and you want to save time during job analysis by defining the minimum required levels: the minimum required level and one level above that to establish categories as required by category rating rules. If the hiring action was for a GS13 position, the “meets” level would mean the applicant needs 52 weeks experience at that level in order to qualify for GS13. If your hiring action is for two GS grades, you will need at least three proficiency levels.

Reviewer: ```suggestion
Use two levels if your announcement is only for one GS grade and you want to save time during job analysis by defining the minimum required levels: the minimum required level ("meets") and one level above that ("exceeds") to establish categories as required by category rating rules. If the hiring action was for a GS-13 position, the “meets” level would mean the applicant needs 52 weeks experience at that level in order to qualify for GS-13. If your hiring action is for two GS grades, you will need at least three proficiency levels.
```

#### Example (for an analytical competancy)

* **GS-13 Meets:** Breaks problems down into component parts. Displays repeated
experience in qualitative and quantitative analysis. Defines product or project metrics
beyond the basics and ties these metrics to decisions.
* **GS-13 Exceeds:** Brings analytical thinking to everything they do, and has a range and depth of
experience doing so. Anticipates the need for metrics and analysis early in the product
and project discovery and design process, and carries metrics through iterations.

### Four levels

Use four levels when you may want to leverage this work for other grades in the future; the same set of competencies and proficiencies can be used for different grade levels because roles that require more seniority set the required proficiencies higher than junior roles. It may also help to define four levels before deciding which level is appropriate for the grade in question. If you use four levels, we recommend these titles for each level:

1. Novice
2. Competant

> 3. Advanced

Reviewer: ```suggestion
3. Advanced - the minimum level required for the grade you're currently hiring for
```

Reviewer 2: not sure about this - sometimes "competent" will be the minimum.

Reviewer: Then I would change below where it says "Advanced (required)" in the example

Maybe just add some language at the top that says one of those two should act as your minimum required competency for the grade level you're currently hiring at?

Reviewer 2: ohh - that was for the specific "you are using these four things in your hiring action" - the intent is to show how the four and the two can mean the same thing (the four just has extra). I can explain this more.

4. Expert

> Once proficiency levels are defined, HR and the SMEs will decide at which level you would require applicants to have one year of experience in order to qualify for the grade in question. For example, they made decide that to qualify for a GS13, applicants must have one year experience at the “Advanced” level. See this example:

Reviewer: ```suggestion
Once proficiency levels are defined, HR and the SMEs will decide at which level you would require applicants to have one year of experience in order to qualify for the grade in question. For example, they made decide that to qualify for a GS-13, applicants must have one year of experience at the “Advanced” level. See this example:
```

#### Example (for an analytical competancy)

> * **Novice:** Unable to break down problems or only talks through problems at a high

Reviewer: @wslack -- How could this level, as defined, be helpful in a future action? Would we want to hire a GS-11 who actively eschews data in decision-making?

Reviewer 2: the four competency model makes you write  down what someone unqualified looks like, which in theory helps to sharpen the others. 

We can switch it back to "collaboration" but I'm also good using something else from the library (this is what's in the [library's explanation](https://smeqa.usds.gov/toolkit/job-analysis/sample-competencies-proficiencies.pdf))

Reviewer: Right, but I think it's unqualified for a GS-13... not unqualified for government work in general. I think actively eschewing data in decision making is like a critical fail. So yeah, I think "collaboration" is a good change.

Reviewer 2: k

> level. Does not make data-driven decisions, or actively eschews data in decision-making.

Reviewer: ```suggestion
level. Does not make data-driven decisions, or actively eschews collaboration in decision-making.
```

Reviewer 2: rejecting this because the collaboration text is different across the board

experience in qualitative and quantitative analysis. Defines product or project metrics
beyond the basics and ties these metrics to decisions.
* **Expert:** Brings analytical thinking to everything they do, and has a range and depth of
experience doing so. Anticipates the need for metrics and analysis early in the product
and project discovery and design process, and carries metrics through iterations.

### Generic proficiency levels

**Novice:** Applies the competency in simple situations, requiring close and extensive guidance. Has the level of experience gained in a classroom and/or experimental scenarios or as a trainee on-the-job. Aware of terminology, concepts, principles, and issues related to this competency.

> Novices have had little experience of the situation in which they are expected to perform. Their initial learning about the situation is in terms of objective attributes--those that are measurable. These are features of the task world that can be recognized without situational experiences. Novices are limited in their understanding to context-free rules that guide action--this means their behavior is limited and inflexible.

> **Basic:** Applies the competency in somewhat difficult situations, requiring frequent guidance. Familiar with and can discuss terminology, concepts, principles, and issues related to this competency.

Reviewer: @wslack -- "Basic" is not a level identified above. Why are we adding a new level between "Novice" and "Competent?" I also really don't love the term "Basic" here, as it's very generic.

Reviewer 2: The NIH model and the other I linked both have 5 levels. I don't like "basic" either and we can totally pull it if you want. Alternatively, we could cut "novice" since I don't think anyone will ever use it to qualify someone and start with this level, though I might want to rename it. (NIH said "basic" - the other said "advanced beginner"

Reviewer: Agreed. Pulling Basic and matching the 4-level model above. Showing a 2-level, 4-level, and 5-level model on the same page is just too much complexity.

> **Basic:** Applies the competency in somewhat difficult situations, requiring frequent guidance. Familiar with and can discuss terminology, concepts, principles, and issues related to this competency.

Reviewer: ```suggestion
```

> > Basic performers have coped with enough real situations to note (or have pointed out to them) recurring, meaningful situational components. At this level, understanding of aspects of the situation is limited to global characterisitics that reflect prior experience in actual situations. The basic performer needs help setting priorities, because they operate on general guidelines and are only beginning to perceive recurrent, meaningful patterns.

Reviewer: ```suggestion
```

**Competent:** Applies the competency in difficult situations, requiring occasional guidance. Able to successfully complete tasks in this competency as requested. Demonstrates understanding of terminology, concepts, principles, and issues related to this competency.
 

> > Competant performers can see their actions in terms of long-range goals or plans. They are consciously aware of formulating, evaluating, and modifying goals-plans. The competent performer is able to generate plans in terms of current and contemplated future aspects that are most important, and those that are not. The competent performer lacks the speed and flexibility that emerges at higher levels of expertise but has a sense of mastery and the ability to cope with and manage a variety of situations.

Reviewer: ```suggestion
> Competent performers can see their actions in terms of long-range goals or plans. They are consciously aware of formulating, evaluating, and modifying goals or plans. The competent performer is able to generate plans in terms of current and contemplated future aspects that are most important, and those that are not. The competent performer lacks the speed and flexibility that emerges at higher levels of expertise but has a sense of mastery and the ability to cope with and manage a variety of situations.
```

**Advanced:** Applies the competency in considerably difficult situations, generally requiring little or no guidance. Recognized within immediate organization as "a person to ask" when difficult questions arise regarding this skill. Demonstrates broad understanding of terminology, concepts, principles, and issues related to this competency.

> > Advanced performers perceive situations as wholes, rather than in terms of situational components. Their performance is guided by "maxims." Perception is key. The perspective is NOT thought out but "presents itself" based upon experience. The proficient performer has learned what typical events to expect in a given situation and how plans need to be modified in accord with these events. This also means that she or he can recognize when the expected typical picture does not materialize and can modify plans and goals accordingly. Situational aspects stand out as more or less important in *this* situation.

Reviewer: ```suggestion
> Advanced performers perceive situations as wholes, rather than in terms of situational components. Their performance is guided by "maxims." Perception is key. Perspectives on common problems do not require significant time spent thinking things through, rather the solution "presents itself" based upon accumulated experience. The proficient performer has learned what typical events to expect in a given situation and how plans need to be modified in accord with these events. This also means that they can recognize when the expected typical picture does not materialize and can modify plans and goals accordingly. Situational aspects stand out as more or less important in *this* situation.
```

Reviewer: @wslack -- Generally speaking, use 3rd person gender-neutral "they" whenever possible, avoiding deciding whether to put "he" or "she" first, and including non-binary gender identities.

Reviewer 2: Yep, I should have caught that beyond the pronoun order. Thank you.

**Expert:** Applies the competency in exceptionally difficult situations, serving as a key resource and advising others. Demonstrates comprehensive, expert understanding of terminology, concepts, principles, and issues related to this competency.
 

> > Expert performers no longer rely on analytic principles (rules, guidelines, maxims) to connect their understanding of the situation to an appropriate action. The expert, with an enormous background of experience, has an intuitive grasp of each situation and zeros in on the accurate region of the problem without wasteful consideration of a large range of unfruitful, alternative diagnoses and solutions. The performer is no longer aware of features and rules, and his or her performance becomes fluid and flexible and highly proficient.

Reviewer: ```suggestion
> Expert performers no longer rely on analytic principles (rules, guidelines, maxims) to connect their understanding of the situation to an appropriate action. The expert, with an enormous background of experience, has an intuitive grasp of each situation and zeros in on the accurate region of the problem without wasteful consideration of a large range of unfruitful, alternative diagnoses and solutions. The performer is no longer aware of features and rules, and their performance becomes fluid and flexible and highly proficient.
```
